Podfile

```diff
@@ -3,9 +3,9 @@ use_frameworks!
 inhibit_all_warnings!
 
 target 'WalkingChallenge' do
-  pod 'FacebookCore', '~> 0.3.0'
-  pod 'FacebookLogin', '~> 0.3.0'
-  pod 'FacebookShare', '~> 0.3.0'
+  pod 'FacebookCore', '~> 0.5.0'
+  pod 'FacebookLogin', '~> 0.5.0'
+  pod 'FacebookShare', '~> 0.5.0'
   pod 'SnapKit', '~> 4.0.0'
```

Comment:
While we're updating pods how about updating snap kit to `4.2.0`?

Reply:
Makes sense, I was being judicious about what was being updated to avoid adding additional scope, but it doesn't look like there are any breaking changes in the latest version of SnapKit, so will bump